lib/Doctrine/DBAL/Driver/AbstractPostgreSQLDriver.php

```diff
@@ -31,7 +31,9 @@ abstract class AbstractPostgreSQLDriver implements Driver, ExceptionConverterDri
      */
     public function convertException($message, DriverException $exception)
     {
-        switch ($exception->getSQLState()) {
+        $sqlState = $exception->getSQLState();
+
+        switch ($sqlState) {
```

Comment:
It looks like this was not an essential part of the fix. `$sqlState` is only used by the `switch` statement anyway. But creating a local variable will do no harm.

Reply:
@phil-davis, you're right. It was introduced in the original version of this PR (https://github.com/doctrine/dbal/commit/5a7479c29481949c92784850c164411b328f527c) which then had to be reworked. This chunk could have been reverted if noticed earlier.